tools/dependencies.sh

```diff
@@ -8,8 +8,6 @@
 # information across all those.
 
 PG_VERSIONS='12 13 14 15'
-# TODO: remove this once TimescaleDB supports PostgreSQL 15: issue #648
-TSDB_PG_VERSIONS='12 13 14'
```

Comment:
Let's hold off removing this until all references are gone in this repository and in release-scripts.